### PR TITLE
Better authinfo example

### PR DESCRIPTION
**Pull request by elhu**
This PR adds a more explicit example on how to provide data regarding the user, notably:
- `userName`
- `userIdentifier`
- `authenticationInfo`

To be noted: I had to change the declaration of the background registration task (`app/src/main/java/com/dimelo/sampleapp/MainActivity.java:117`) for the app to compile. This should probably be tested in another environment before being merged in master.

### CODE REVIEW

File: app/src/main/java/com/dimelo/sampleapp/MainActivity.java [hunk 1]
```diff
@@ -68,11 +74,28 @@ else if (error.statusCode == DimeloConnection.DimeloError.TIMEOUT_ERROR){
...
         Dimelo.setup(this);
         Dimelo dimelo = Dimelo.getInstance();
         dimelo.setApiSecret(secret);
+//        dimelo.setUserIdentifier("42");
+        dimelo.setUserName("John Doe");
+
+        JSONObject authInfo = new JSONObject();
+        PackageInfo pInfo = null;
+        try {
+            pInfo = getPackageManager().getPackageInfo(getPackageName(), 0);
```
Comment (nel) [inline app/src/main/java/com/dimelo/sampleapp/MainActivity.java]: Well I'd prefer a less convoluted example as this is a sample app (no package info).

But we should add that in the header of our requests.

Customer_id is a simpler example, but keep Dimelo rocks :)

**Comment by nel** [comment]
Good to go but with a more educational and simple authenticationInfo

File: app/src/main/java/com/dimelo/sampleapp/MainActivity.java (hunk 1 above)
Comment (elhu) [inline app/src/main/java/com/dimelo/sampleapp/MainActivity.java]: I've cleaned up the `setAuthenticationInfo` example, with an arbitrary `customerId` field, and I kept Dimelo Rocks :).

File: app/src/main/java/com/dimelo/sampleapp/MainActivity.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
 package com.dimelo.sampleapp;
 
 import android.content.Context;
+import android.content.pm.PackageInfo;
```
Comment (nel) [inline app/src/main/java/com/dimelo/sampleapp/MainActivity.java]: Not needed anymore

File: app/src/main/java/com/dimelo/sampleapp/MainActivity.java [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
+import android.content.pm.PackageManager;
```
Comment (nel) [inline app/src/main/java/com/dimelo/sampleapp/MainActivity.java]: Not needed anymore

Comment (elhu) [inline app/src/main/java/com/dimelo/sampleapp/MainActivity.java]: Fixed. I don't know why, I thought Android Studio took care of that by itself.

**Pull request merged**